persistence-cassandra-3.11/src/test/java/io/stargate/db/cassandra/impl/CassandraPersistenceIT.java

```diff
@@ -50,11 +64,13 @@ public static void createPersistence(ClusterConnectionInfo backend) throws IOExc
     }
 
     persistence = new CassandraPersistence();
-    persistence.initialize(makeConfig(baseDir));
+    config = makeConfig(baseDir);
+    persistence.initialize(config);
+    setup(testInfo, backend);
```

Comment:
I removed `@BeforeEach` from `setup()` in PersistenceTest to ensure that it runs after `createPersistence()` in CassandraPersistenceIT. Note that I also had to change the signature of `createPersistence()`.